rir/src/compiler/translations/rir_compiler.h

```diff
@@ -26,10 +26,14 @@ class RirCompiler {
     }
 
     bool isVerbose() { return verbose; }
-    void setVerbose(bool v) { verbose = v; }
+    bool shouldPrintOriginalVersion() { return verbose & 0X1; }
+    bool shouldPrintCompiledVersion() { return verbose & 0X10; }
+    bool shouldPrintOptimizations() { return verbose & 0X100; }
+    bool shouldPrintInliningVersions() { return verbose & 0X1000; }
```

Comment:
all the bit masks are hex numbers, ie. `x & 0x10` will look at the sixteenth bit. I guess you meant to look at the second, which would be `x & 0x2`

Reply:
(Fifth bit, not sixteenth.)

Alternatively, C++14 has binary literals, e.g. `0b10`. Or you could use bitshifts, e.g. `1 << 1`.

Reply:
:). 
Fixed! 
Thanks.